src/components/MovieDetails.tsx

```diff
@@ -0,0 +1,31 @@
+type MovieDetailsProps = {
+    coverUrl: string
+    title: string
+    year: number
+    genres: string[]
+    rating: number
+    duration: number
+    description: string
+}
+
+export function MovieDetails({
+    coverUrl,
+    title,
+    year,
+    genres,
+    rating,
+    duration,
+    description,
+}: MovieDetailsProps) {
+    return (
+        <div>
+            <img src={coverUrl} alt={title} />
+            <h2>{title}</h2>
+            <p>{rating}</p>
+            <p>{genres.join(', ')}</p>
+            <p>{year}</p>
+            <p>{duration}</p>
```

Comment:
As I can see from the prototype, duration should be rendered as hours. So, instead of `120`, `2h 0min` should be rendered.